classy_bot/codeguessr.py

```diff
@@ -76,23 +81,36 @@ def quiz_from_solution(*, solution: Solution, langs: list[str], n_choices: int)
     )
 
 
-def random_quiz_from_db(db_uri: str, *, n_choices: int) -> Quiz:
+def random_quiz_from_db(db_uri: str, *, n_choices: int) -> MultiChoiceQuiz:
     return quiz_from_solution(
         solution=random_solution_from_db(db_uri),
         langs=langs_from_db(db_uri),
         n_choices=n_choices
     )
 
 
-class CodeguessrQuizView(QuizView):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class CodeguessrQuizView(MultiChoiceView):
+    def __init__(
+        self,
+        *,
+        interaction: Interaction,
+        quiz: MultiChoiceQuiz,
+        color: discord.Color | int | None = None,
+        timeout: float = 60
+    ) -> None:
+        super().__init__(
+            interaction=interaction,
+            quiz=quiz,
+            color=color,
+            timeout=timeout
+        )
+
```

Comment:
Personally I prefer the `*args, **kwargs` approach so changes to parameters of the parent class don't force changes in other places in the code. For this it's fine though...